oqs-template/generate.yml

```diff
@@ -0,0 +1,61 @@
+kems:
```

Comment:
What about Kyber, NTRU, Saber?

Reply:
I didn't include them in this PR to avoid cluttering the review, and I'll add them in a separate commit once I put up the PR for the signature templating code.